bmq-sdk/src/main/java/com/bloomberg/bmq/QueueOptions.java

```diff
@@ -314,13 +348,16 @@ public Builder merge(QueueOptions options) {
             if (options.hasSuspendsOnBadHostHealth()) {
                 setSuspendsOnBadHostHealth(options.getSuspendsOnBadHostHealth());
             }
+
+            subscriptions.clear();
```

Comment:
Here we should probably overwrite subscriptions if `options.hasSubscriptions() == true`